OMEdit/OMEdit/OMEditGUI/Annotations/TextAnnotation.cpp

```diff
@@ -159,7 +159,7 @@ void TextAnnotation::parseShapeAnnotation(QString annotation)
   FilledShape::parseShapeAnnotation(annotation);
   // parse the shape to get the list of attributes of Text.
   QStringList list = StringHandler::getStrings(annotation);
-  if (list.size() < 12) {
+  if (list.size() < 15) {
```

Comment:
Consider the code below it should be < 15. However, this is far from a proper solution I think